template/Makefile.tmpl

```diff
@@ -8,6 +8,9 @@ proto:
 	go get github.com/golang/protobuf/protoc-gen-go
 	protoc -I . {{.Name}}.proto --lile-server_out=. --go_out=plugins=grpc:${GOPATH}/src
 
+build: proto
+	go build -o bin/{{.RelativeName}} {{.RelativeName}}/main.go
```

Comment:
in our Dockerfile we currently build to `./build` instead of `bin`, mostly because bin, isn't in most peoples paths? and if you wanted to install your app you'd use `go install` I guess?

If we move this to build I'm happy to also use this command in the Dockerfile too, as then anyone who adds custom build steps will get the same in their Docker

Reply:
sounds fine to me - happy to switch to `build`